src/plugins/config.ts

```diff
@@ -59,6 +60,11 @@ export default fastifyPlugin(
                     type: "string",
                     default: "prod",
                 },
+                ALLOWED_ORIGINS: {
+                    type: "string",
+                    separator: ",",
+                    default: "http://localhost:4321",
```

Comment:
Perhaps update `.env.example` with this addition

Reply:
Since it's not a required config I felt like it wasn't necessary. It'll probably only be changed in Vercel.  
You decide @frederikpyt

Reply:
I don't mind